### ISSUE TITLE
Script Installation Fails

### ISSUE THREAD

**Issue by joesixpack** [Chainflow/solana-mission-control#4]
![image](https://user-images.githubusercontent.com/23145642/129656899-fd1991bd-320a-4160-b3de-aee7c39e9faa.png)

**Comment by PrathyushaLakkireddy** [comment]
Hello @joesixpack, can you check did you set GOBIN path? 
once do `echo $GOBIN` , and see if it's empty or have you configured already? and do you already have installed go?

**Comment by PrathyushaLakkireddy** [comment]
@joesixpack, I have updated installation script, can you please pull that latest updates and run?

**Comment by joesixpack** [comment]
Looks the problem is the script assumes that if go is already installed, then the env variables already exist.  This is not the default case for repository installs of go.

Other issues:

![image](https://user-images.githubusercontent.com/23145642/129975242-d23d4a76-20d2-4871-9d69-1c87981307d8.png)
Wrong path here.  Should point to the raw file not HTML.

The TG Chat ID env variable needs to be in quotes to support negative numbers.

The systemd config file needs to have the Solana path included in an Environment statement.

**Comment by joesixpack** [comment]
The skip rate threshold needs to be user configurable, I'm being flooded with alerts!

**Comment by PrathyushaLakkireddy** [comment]
> Looks the problem is the script assumes that if go is already installed, then the env variables already exist. This is not the default case for repository installs of go.
> 
> Other issues:
> 
> ![image](https://user-images.githubusercontent.com/23145642/129975242-d23d4a76-20d2-4871-9d69-1c87981307d8.png)
> Wrong path here. Should point to the raw file not HTML.
> 
> The TG Chat ID env variable needs to be in quotes to support negative numbers.
> 
> The systemd config file needs to have the Solana path included in an Environment statement.

Script, checks for the go, if it's not installed already, then it will install and configure the go in that system.
Will check the link issue.
It won't be a problem with negative numbers as well and it supports negative numbers as well.

**Comment by PrathyushaLakkireddy** [comment]
> The skip rate threshold needs to be user configurable, I'm being flooded with alerts!

Yeah, at present it was implemented like, if the validator skip rate exceeds network skip rate.
Will check and add the skip rate in config.

Thank you.
